src/pages/Ticket/Seat/TicketInfo/TicketInfo.js

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import styled from 'styled-components';
+import Common from '../../../../components/Common/Common';
+
+const TicketInfo = () => {
+  const { ScreenOut, IcoMovie } = Common;
+
+  return (
+    <GroupSeatInfo>
+      <WrapInfo>
+        <TxtInfo>
+          <ScreenOut>관람 정보 : </ScreenOut>건대입구, 4관 4층 남은좌석 122/127
+        </TxtInfo>
+        <EmphTime>
+          <ScreenOut>관람 시간 : </ScreenOut>2023.03.29 &#40;수&#41; 20:10 ~
+          21:57
+        </EmphTime>
+      </WrapInfo>
+      <WrapSeatInfo>
```

Comment:
```suggestion
      <div>
```

아래 스타일 부분을 보니까 아무런 스타일을 지정하지 않고 영역만 정해주신 것 같네요!
이럴 경우에는 위와 같이 태그를 사용해도 무방합니다!

Reply:
수정했습니다~ 확인 부탁드립니다 감사합니다 :)